source/features/use-first-commit-message-for-new-prs.tsx

```diff
@@ -6,25 +6,38 @@ import * as textFieldEdit from 'text-field-edit';
 import features from '.';
 import looseParseInt from '../helpers/loose-parse-int';
 
+// TODO [2022-05-01]: Drop GHE code and merge function back in init
+function getFirstCommitMessage(): string[] {
+	if (pageDetect.isEnterprise()) {
+		return select('#commits_bucket [data-url$="compare/commit"] a[title]')!.title.split('\n\n');
+	}
+
+	// Linkified commit summaries are split into several adjacent links #5382
+	const commitSummary = select.all('.js-commits-list-item:first-child .js-details-container > p > a')
```

Comment:
This selector isn't great. Maybe we should keep `a.Link--primary` and do a second `select` on its `parentElement`.